akka-http-core/src/main/scala/akka/http/impl/model/parser/HeaderParser.scala

```diff
@@ -62,18 +62,21 @@ private[http] class HeaderParser(
 
   ///////////////// DynamicRuleHandler //////////////
 
-  type Result = Either[ErrorInfo, HttpHeader]
+  type Result = HeaderParser.Result
```

Comment:
I was confused why not just use `HeaderParser.Result` directly, but this is specifying the `Result` from `DeliveryScheme`.

It looks like `type` also accepts an `override` modifier, might be a nice way to make that explicit?

Reply:
:+1: